dolphin-dnn/src/main/proto/neural_network.proto

```diff
@@ -23,6 +23,14 @@ message FullyConnectedLayerConfiguration {
   required uint32 num_output = 4;
 }
 
+message PoolingLayerConfiguration {
+  required string pooling_type = 1;
+  required uint32 stride_height = 2;
+  required uint32 stride_width = 3;
```

Comment:
How about having default values for `stride_height` and `stride_width` as 1? If so, these can be optional.